.teamcity/performance-test-durations.json

```diff
@@ -786,70 +786,28 @@
   "scenario" : "org.gradle.performance.regression.nativeplatform.SwiftCleanBuildPerformanceTest.clean assemble (swift)",
   "durations" : [ {
     "testProject" : "bigSwiftApp",
-    "linux" : 2583
+    "linux" : 2545
   }, {
     "testProject" : "mediumSwiftMulti",
-    "linux" : 3151
-  } ]
-}, {
-  "scenario" : "org.gradle.performance.BuildScanPluginPerformanceTest.with and without plugin application (clean build - 20 projects - slow tasks - less console)",
```

Comment:
🤔 @wolfs is it expected, that `BuildScanPluginPerformanceTest` no longer run since ~ December 17-18?

Reply:
I am actually surprised that it is in there - the test currently runs without using the scenario things at all. It shouldn't have the scenario annotation either.

I think it stopped running, since we optimized the pipeline on the Develocity side - nobody has been paying attention to the test anyway. Thanks for notifying me!

Reply:
I'll accept the removal of the related data then and merge this PR. Thanks!